cips/cip-6.md

```diff
@@ -4,7 +4,8 @@ title: Price enforcement
 description: Enforce payment of the gas for a transaction based on a global minimum price 
 author: Callum Waters (@cmwaters)
 discussions-to: https://forum.celestia.org/t/cip-006-price-enforcement/1351
-status: Review
+status: Last Call
+last-call-deadline: 30-01-2024
```

Comment:
ensure it follows a YYYY-MM-DD format

Reply:
c1b6458e9465c88a500d5276c510b58cd6c1f1e4